.github/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+*   @appetrosyan
+
+# doc sources
+/src/   @outoftardis @WRRicht3r @appetrosyan
+*.md    @outoftardis @WRRicht3r @appetrosyan
+src/guide/javascript.md @0x009922 @outoftardis @WRRicht3r @appetrosyan
```

Comment:
There are some Vue components (`*.vue`) alongside Markdown files too. I think I am responsible for them.

Reply:
added a line fir this